### PR TITLE
add tips for advanced debugging

### PR DESCRIPTION
**Pull request by jaimergp**
### Checklist - did you ...

- [x] Add a file to the `news` directory ([using the template](../blob/main/news/TEMPLATE)) for the next release's release notes?
     
- [ ] Add / update necessary tests?
- [X] Add / update outdated documentation?

Comes from https://github.com/conda/menuinst/issues/178

### CODE REVIEW

File: docs/source/cli-options.md [hunk 1]
```diff
@@ -81,13 +101,45 @@ Run the uninstaller in silent mode from its original location:
 > cmd.exe /c start /wait "C:\Program Files\my_app\uninstaller.exe" /S _?=C:\Program Files\my_app
 ```
 
+:::{admonition} EXE installers with file logging
+:class: tip
+
+Windows installers do not have a verbose mode. By default, the graphical logs are only available in the "progress bar" dialog, by clicking on "Show details". This text box is right-clickable, which will allow you to copy the contents to the clipboard (and then paste them in a text file, presumably).
+
+There's a way of building EXE installers that do write logs to a file. For this, you need a special `nsis` package configured to do so:
```
Comment (beeankha) [inline docs/source/cli-options.md:109]:
```suggestion
There's a way of building EXE installers that can write logs to a file; for this, you need a special `nsis` package configured to do so:
```

File: docs/source/cli-options.md [hunk 2]
```diff
@@ -81,13 +101,45 @@ Run the uninstaller in silent mode from its original location:
...
+
+```batch
+> conda install "nsis=*=*log*"
+```
+
+Then, you can invoke `constructor` normally, but setting a special environment variable:
```
Comment (beeankha) [inline docs/source/cli-options.md:115]:
```suggestion
Then, you can invoke `constructor` normally after setting a special environment variable:
```

**Comment by jaimergp** [comment]
Thanks @beeankha!

File: docs/source/cli-options.md [hunk 3]
```diff
@@ -32,22 +38,36 @@ Run the installer in batch mode and install to a custom path:
...
+  variable. The default is `0`. This is NOT recommended.
+- `/CheckPathLength=[0|1]`: Check whether the installation path is too long (>46
+  characters). The default depends on how the installer was created.
+- `/KeepPkgCache=[0|1]`: Whether to keep the package cache or not. Defaults to `1`.
+- `/NoRegistry=[0|1]`: Whether to prevent registry modification or not. Defaults to `0`.
+- `/NoScripts=[0|1]`: If set to `1`, the installer will not run any post-install scripts. Defaults
+  to `0`.
+- `/NoShortcuts=[0|1]`: If set to `1`, the installer will not create any shortcuts. Defaults to
+  `0`.
+- `/RegisterPython=[0|1]`: Whether toregister Python as default in the Windows registry. Defaults
```
Comment (beeankha) [inline docs/source/cli-options.md:69]:
```suggestion
- `/RegisterPython=[0|1]`: Whether to register Python as default in the Windows registry. Defaults
```

File: docs/source/debugging.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+
+## Verbose `conda`
+
+Whether it's while running `constructor` to build an installer, or while the installer is running in the target machine, some instance of `conda` will be running behind the scenes. You can request more verbose output via the `CONDA_VERBOSITY` environment variable. It can take values from `1` to `4`. You can also set `CONDA_DEBUG` to `1`.
+
+
+## Verbose shell installers
+
+The shell installers are "just" a shell script that invokes `conda` in certain events. This means you can enable `bash` verbosity via the `-x` flag:
```
Comment (beeankha) [inline docs/source/debugging.md:20]:
```suggestion
The shell installers are simply shell scripts that invoke `conda` in certain events. This means you can enable `bash` verbosity via the `-x` flag:
```

File: docs/source/debugging.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+$ bash -x ./Miniconda.sh
+# or with verbose conda:
+$ CONDA_VERBOSITY=3 bash -x ./Miniconda.sh
+```
+
+## Verbose PKG installers
+
+PKG installers are usually invoked graphically. You can check the native logs via <kbd>⌘</kbd>+<kbd>L</kbd>. Note that you will need to choose the detailed view in the dropdown menu you'll find in the top right corner.
+
+In order to get more verbosity out of conda, you now know you need the `CONDA_VERBOSITY` variable. However, it needs to be set up _before_ running the installer. One way from the command line would be:
```
Comment (beeankha) [inline docs/source/debugging.md:32]:
```suggestion
In order to get more verbosity out of `conda`, you now know you need the `CONDA_VERBOSITY` variable. However, it needs to be set up _before_ running the installer. One way from the command line would be:
```

Typically we only monospace "conda" if it's referred to as a command, but it seems like constructor docs monospace both "`constructor`" and "`conda`" so I'm suggesting this change here for the sake of consistency

File: docs/source/debugging.md [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+If you want `conda` to print more details, then, run it from the CMD prompt like this:
+
+```batch
+> set "CONDA_VERBOSITY=3"
+> cmd.exe /c start /wait your-installer.exe
+```
+
+### Building logging-enabled EXE installers
+
+There's a way of building EXE installers that do write logs to a file. For this, you need a special `nsis` package configured to do so:
```
Comment (beeankha) [inline docs/source/debugging.md:53]:
```suggestion
There's a way of building EXE installers that can write logs to a file; for this, you need a special `nsis` package configured to do so:
```

File: docs/source/debugging.md [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+
+```batch
+> conda install "nsis=*=*log*"
+```
+
+Then, you can invoke `constructor` normally, but setting a special environment variable:
```
Comment (beeankha) [inline docs/source/debugging.md:59]:
```suggestion
Then, you can invoke `constructor` normally after setting a special environment variable:
```

**Review by beeankha — APPROVED** [review]
Just some minor edits, otherwise LGTM!

**Review by dholth — APPROVED** [review]

**Review by beeankha — APPROVED** [review]

**Pull request merged**